### DISCUSSION TITLE
Does prism support Maui?

### DISCUSSION THREAD

**Discussion by BoyFaceGirl** [PrismLibrary/Prism#2595] [.NET MAUI] [answered]
Prism is great and Maui is worth looking forward to. I wonder if prism will support the Maui platform？

**Comment by brianlagunas** [comment]
It will

**Reply by ederbond** [reply]
Eagerly waiting for this.

**Comment by dansiegel** [comment, marked as answer]
There is a preview package that was built for .NET 6 preview 4. The project where we are playing with this and will release another release at some point is https://github.com/dansiegel/prism.maui

There is currently an issue with the navigation which is why I haven't released an update. But if you'd like to contribute it would help speed up its release.

**Comment by developer9969** [comment]
@dansiegel @brianlagunas Hi there we would like to start moving to MAUI but bit stuck as we are using Prism.
Is there any plan to have some sort of Beta out  ?

Many thanks
